components/vs-code-extension/code/vs-code-extension-bbe/ballerina-copilot.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Ballerina Copilot"
+description: "
+
+Ballerina Copilot seamlessly integrates AI-driven capabilities into your integration developement workflow, transforming natural language into high-quality Ballerina code. Boost your productivity with powerful features designed specifically for Ballerina developers, including:
+
+-   AI-Powered Code Generation: Write complex Ballerina code effortlessly using simple, natural language prompts.
```

Comment:
These need to be in sentence case,
AI-Powered Code Generation -> AI-powered code generation